module/People/src/People/Service/OrganizationCommandsListener.php

```diff
@@ -22,12 +22,23 @@ protected function onOrganizationCreated(StreamEvent $event) {
 	}
 	
 	protected function onOrganizationUpdated(StreamEvent $event) {
+		$id = $event->metadata()['aggregate_id'];
+		$entity = $this->entityManager->find(Organization::class, $id);
+		$entityUpdated = false;
 		if(isset($event->payload()['name'])) {
-			$id = $event->metadata()['aggregate_id'];
-			$entity = $this->entityManager->find(Organization::class, $id);
-			$updatedBy = $this->entityManager->find(User::class, $event->payload()['by']);
-				
 			$entity->setName($event->payload()['name']);
+			$entityUpdated = true;
+		}
+		if(isset($event->payload()['kanbanizeColumnMapping'])) {
+			$entity->setSetting('kanbanizeColumnMapping', $event->payload()['kanbanizeColumnMapping']);
+			$entityUpdated = true;
+		}
+		if(isset($event->payload()['kanbanizeAccountAddress'])) {
```

Comment:
Lo rinomino in 'AccountSubdomain'; contiene un'informazione sbagliata, cioè tutto l'indirizzo di chiamata alle API, invece che solamente il subdomain relativo all'account di Kanbanize. Lo correggo.